spec/factories/profiles.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :profile do
+    first_name { "First Name" }
```

Comment:
You might want to use `Faker` to generate unique names for each test. Having the same name for every test could cause weird test failures.